.env.example

```
@@ -1 +1,2 @@
-COMMIT_SHA=test
\ No newline at end of file
+COMMIT_SHA=test
+HYPER_API_TOKEN=41e50e69-6da4-4462-b70e-c7b5e7b70f05
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)